Generator/generator-botbuilder-python/generators/app/templates/requirements.txt

```diff
@@ -0,0 +1,5 @@
+git+https://github.com/Azure/msrest-for-python@async2
+botbuilder-schema>=4.0.0.a2
```

Comment:
@johnataylor is right, you should be able to just install botbuilder-core and it'll pull its dependencies down without running into any issues.

Reply:
So we don't need this file then?

Reply:
No, the file should be included, but we probably don't need `botbuilder-schema` and `botframework-connector` as there is a dependency on `botbuilder-core`.

Reply:
Got it. I think I had a chat about this with John, I remember now.